engines/batch-ensembler/tests/sink_test.py

```diff
@@ -26,7 +26,7 @@ def test_load_from_config(sink_config):
 
     assert sink.type == sdk.sink.BigQuerySink.TYPE
     assert isinstance(sink, BigQuerySink)
-    assert sink.save_mode == sdk.sink.SaveMode.OVERWRITE.to_open_api()
+    assert sink.save_mode.name == sdk.sink.SaveMode.OVERWRITE.to_open_api().to_str()
```

Comment:
Should it be `assert sink.save_mode == sdk.sink.SaveMode.OVERWRITE` ?

Reply:
Yeah. This is hilarous.